### ISSUE TITLE
Oh no! It's broken

### ISSUE THREAD

**Issue by cmoyer2** [pivotal/postfacto#132]
Installed everything to PCF and it does not appear any installation errors occurred.  When trying to visit the home page, get the error page "Oh no! It's broken".  Both the web and api app are running up and running in PCF, API app has attached to both MySQL and Redis services appropriately.  The only error I see from the client side is most resources are retrieved correctly except for the config, which is trying to retrieve the config from the following API instead of the mapped PCF API URL.  Did I get something incorrect on the install?

Request URL: https://postfacto-api.%7B%7Bpcf-url%7D%7D/config

Install command was:

>./deploy.sh postfacto postfacto-api {my_pcf_url}

**Comment by MichaelDBrown** [comment]
Same thing happening to me, log errors are about unable to create websocket.

**Comment by MichaelDBrown** [comment]
The error from the api is "Request origin not allowed" when trying to upgrade the websocket. I've gone back to 2.6.1 and I'm getting the same problem.

**Comment by MichaelDBrown** [comment]
So the problem is with the cf-url that gets passed to the deployment script. If the deploy.sh does a login, it assumes that the domain matches where the ui is going to run from. That's not always the case. To solve the problem, I modified the deploy.sh script to NOT login to cf as I'm already logged in. Then the cf url I passed is the part of the URL that goes behind the web ui URL. If you look in the api manifest, you'll see where the origin header is set and where the assumptions from cf url are happening.  ALSO....the default value that gets added will be wrong since it has https in it but that isn't want you want when it gets used by the manifest.
